src/metabase/api/card.clj

```diff
@@ -362,11 +363,11 @@
   "Compute a 'magic' cache TTL time (in seconds) for QUERY by multipling its historic average execution times by the `query-caching-ttl-ratio`.
    If the TTL is less than a second, this returns `nil` (i.e., the cache should not be utilized.)"
   [query]
-  (when-let [average-duration (qputil/query-average-duration query)]
+  (when-let [average-duration (query/average-execution-time-ms (qputil/query-hash query))]
```

Comment:
you want to create dashboards based on the data in the application database? Are you going to add the application DB as a DB in the admin panel?